boards/mips-malta/include/periph_cpu.h

```diff
@@ -0,0 +1 @@
+/* This file must exist to get timer code to build */
```

Comment:
This file should go into the cpu directory and contain the MCU's peripheral configuration